content/docs/User guides/adding-removing-limits.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Adding or removing limits"
+linkTitle: "Adding or removing limits"
+date: 2019-03-26
+weight: 1
+description: >
+  Add or remove limits.
+---
+
+This guide describes how to add or remove limits, a functionality that allows
+users to specify the maximum number of concurrently running frames 
+associated with that Limit.
+
+
+## Adding or removing a limit
+
+To add or remove a limit:
+
+1.  Open CueGui
+
+1. Load the **Monitor Jobs** view from the **Views/Plugins->Cuetopia** menu.
+
+1. Double click on a job to load its layers.
+
+1. Right click on the job you'd like to modify limits for.
+
+1. Click **Properties**.
+
+1. Check/Uncheck limits you'd like enable/disable for this layer.
```

Comment:
instead of the slashes, I recommend expanding to 'or', especially for users making use of screen readers:

Check or uncheck limits you'd like to enable or disable for this layer.